Updated setting_up_xr.rst to include C# example (#6884)
godotengine/godot-docs: tutorials/xr/setting_up_xr.rst

		else:
			print("OpenXR not initialised, please check if your headset is connected")

  .. code-tab:: csharp

>     public partial class XRInitializer : Node3D

Reviewer: Since the root node in the scene is called `Main`, I think this class should use the same name.

```suggestion
    public partial class Main : Node3D
```

    {
        public override void _Ready()
        {
            XRInterface xrInterface = XRServer.FindInterface("OpenXR");
            if (xrInterface != null && xrInterface.IsInitialized())
            {

>                 GetViewport().UseXR = true;
> 
>                 GD.Print("OpenXR Initialized Successfully!");

Reviewer: The GDScript example has these lines in a different order. We should try to be consistent.

```suggestion
                GD.Print("OpenXR initialized successfully");

                GetViewport().UseXR = true;
```

            }

>             else
>                 GD.Print("OpenXR failed init... check headset connection!");

Reviewer: Use the same message as the GDScript example. Also, our preferred style is to use curly brackets even when not necessary.

```suggestion
            else
            {
                GD.Print("OpenXR not initialized, please check if your headset is connected");
            }
```
